### PR TITLE
Added webpack config

### PR DESCRIPTION
**Pull request by katerinastolyarova**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
```
Comment (elkinuk) [inline .eslintrc]: let's add https://www.npmjs.com/package/eslint-plugin-react here

File: package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "url": "https://github.com/katerinastolyarova/reactJS-mentoring/issues"
+  },
+  "homepage": "https://github.com/katerinastolyarova/reactJS-mentoring#readme",
+  "dependencies": {
+    "clean-webpack-plugin": "^4.0.0",
+    "cross-env": "^7.0.3",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^27.3.1",
+    "react": "^17.0.2",
+    "webpack": "^5.64.1"
```
Comment (elkinuk) [inline package.json]: Please, clean your dependencies, I believe the only packages that should stay are react and cross-env
everything else is devDependency

**Review by elkinuk — CHANGES_REQUESTED** [review]
good job

**Comment by elkinuk** [comment]
Not sure if husky is working
can you show the screenshot of the console?

Screenshot 2021-11-23 at 18 20 48

File: .eslintrc (hunk 1 above)
Comment (katerinastolyarova) [reply .eslintrc]: It was added, thanks

File: package.json (hunk 2 above)
Comment (katerinastolyarova) [reply package.json]: Fixed

**Comment by katerinastolyarova** [comment]
> Not sure if husky is working can you show the screenshot of the console?
> 
> Screenshot 2021-11-23 at 18 20 48

For me it looks like

image

**Pull request merged**